scripts/build.py

```diff
@@ -270,7 +276,13 @@ def _compare_file_count(first_dir_path, second_dir_path):
         ValueError: The source directory does not have the same file count as
             the target directory.
     """
-    first_dir_file_count = get_file_count(first_dir_path)
+
+    first_dir_file_count = 0;
+    if isinstance(first_dir_path, list):
```

Comment:
This seems dodgy. first_dir_path should not be a list, it should be a path -- given the variable name.

Reply:
Yes, fixed.